java/core/src/main/java/sleeper/core/properties/table/TablePropertiesProvider.java

```diff
@@ -61,9 +62,11 @@ protected TablePropertiesProvider(
 
     /**
      * Retrieves properties by table name.
+     * *
      *
-     * @param  tableName the name of the table
-     * @return           the table properties
+     * @param  tableName              the name of the table
+     * @return                        the table properties
+     * @throws TableNotFoundException if the table for the given name is not found.
```

Comment:
This description doesn't need a full stop, as it's not a sentence.

Should this be "with the given name" rather than "for the given name"?